### PR TITLE
Modularize es6

### LINKED ISSUE #1: Modularize jsQUEST & numeric.js
Hey @kurokida,

Continuing one of the topics that came up in our chat on the [PsychoPy forum](https://discourse.psychopy.org/t/jsquest-a-bayesian-adaptive-psychometric-method-for-measuring-thresholds-in-online-experiments/23115): how to turn jsQUEST into a module (and have it play nice with numeric.js)? 

An advantage of numeric.js not being developed anymore is that we can edit its structure without worrying what happens upstream. I'm quite sure I can help you out converting it into a module, but before we go do that, a question...

What kind of module would you like? If I may make a recommendation: ES6 is the most modern standard. A build chain could look a bit like this:
* At PsychoJS we use [babel](https://babeljs.io/) to compile ES6 into JavaScript that is suitable for older browsers. 
* Probably babel can also export ES6 to a global variable, similar to what you've got now, but for sure this can be done with browsersify, as I did in [this experiment](https://gitlab.pavlovia.org/tpronk/demo_embed_html).

Note that I'm not really an expert in build chains, but we've got an expert in our team; I'll pull him in once we've got an outline of our plan.

Best, Thomas

### PR DESCRIPTION
**Pull request by tpronk**
Hey Daiichiro,

This is the modularized version of jsQUEST. This PR closes #1.

Best, Thomas

NB. Note the "closes #1". This makes merging this pull request automatically resolve the issue.

### CODE REVIEW

**Comment by kurokida** [comment]
Thank you very much for sending a pull request.

I have a few questions.
I am new to merge, so I guess I don't understand the rudiments of it.
I would be very happy if you could tell me.

Question 1:
If I merge your request, The README.md is probably overwritten by yours.
Your READEME file is of course useful, but I would also like to keep what is written in my READEME.
What should I do in such a situation?
Once I have merged your request, should I edit the file?

Question 2:
My jsQuestDemo.html seems to link to your dist/demo.html.
How did you create this link?
I'm interested in how you do that.

Question 3: 
If I merge your request, will I lost docs/index.md and the files included in the jsPsychDemo folder?

**Comment by tpronk** [comment]
Hey again!
1. Hmmm, not sure actually. This [page](https://dev.to/codypearce/how-to-add-commits-to-another-person-s-pr-on-your-github-repo-6np) describes how to add your own commits to my PR. Perhaps an easier route is to merge my PR and then make modifications to the README.md?
2. I'm not sure what you mean, can you give me another pointer?
3. It will still be in the history of your repo (via git you can go back to any prior state). See the history of jsQUEST [here](https://github.com/kurokida/jsQUEST/commits/main). Another way is making a new branch.

**Comment by kurokida** [comment]
Thank you for your reply.

I've read the page you provided and understand most of it, but first I'm going to merge your pull request and edit it.

I'm sorry for the lack of explanation on question 2.
On GitHub, it is displayed as shown in the attached figure. (jsQuestDemo.html -> dist/demo.html)
スクリーンショット 2021-06-08 104428

I'm interested in whether the relationship between these two files was created automatically or manually.
In other words, it is not treated as that the jsQuestDemo.html was removed, and the demo.html was added. Did GitHub automatically recognize that the jsQuestDemo.html has been renamed to demo.html?

**Comment by kurokida** [comment]
Thank you for your reply.
I will try to merge your request now.
